specification/billing/resource-manager/Microsoft.Billing/stable/2020-05-01/billing.json

```diff
@@ -4782,6 +4782,14 @@
           "type": "string",
           "readOnly": true
         },
+        "tags": {
```

Comment:
What is the difference between tags and labels?

Reply:
They are the same. Plan was to eliminate one, but now both are being returned and this is causing s360 to flag this. So to fix this issue in GA, i am adding tags also to the swagger spec.